bin/cli.js

```diff
@@ -41,6 +41,11 @@ var _ = require('lodash'),
       bucket: {
         desc: 'bucket name required for templates larger than 50k',
       },
+      context: {
```

Comment:
I can see this being `template` or `templatePath` but I found it confusing since arg 0 is already path / template itself.